### PR TITLE
Fix for long single-word app titles in top-navigation

### PR DESCRIPTION
**Pull request by gd2910**
Fixed a bug where a long single-word app title would push other elements such as 'search' and 'chips' outside of the browser window.

## Summary of the changes
Class added to app title that enables wrap for longer titles

## Related issue
Ticket: https://github.com/mi6/ic-ui-kit/issues/175

## Checklist
- [ ] I have added relevant unit and visual regression tests.
- [x] I have manually accessibility tested any changes, if relevant.
- [ ] I have ensured any changes match the Figma component library.

### CODE REVIEW

File: packages/web-components/src/components/ic-top-navigation/ic-top-navigation.stories.mdx [hunk 1]
```diff
@@ -590,96 +284,60 @@ import NavigationGroup from "../ic-navigation-group/readme.md";
...
-        >
-          Sunrise theme
-        </ic-button> `
+        <ic-button variant="primary" id="default-btn" onClick="document.querySelector('ic-theme').color='rgb(27, 60, 121)'"> Default theme </ic-button>
+        <ic-button variant="primary" id="custom-btn" onClick="document.querySelector('ic-theme').color='rgb(255, 201, 60)'"> Sunrise theme </ic-button> `
+    }
+  </Story>
+</Canvas>
+
+### Long single-word app title
```
Comment (GCHQ-Developer-847) [inline packages/web-components/src/components/ic-top-navigation/ic-top-navigation.stories.mdx:319]: I think we want this same story adding to the React storybook as well (react/src/stories)

File: packages/web-components/src/components/ic-top-navigation/ic-top-navigation.stories.mdx [hunk 2]
```diff
@@ -1,36 +1,21 @@
...
-  Canvas,
-  ArgsTable,
-  Description,
-} from "@storybook/addon-docs";
-import { html } from "lit-html";
-import readme from "./readme.md";
-import NavigationButton from "../ic-navigation-button/readme.md";
-import NavigationItem from "../ic-navigation-item/readme.md";
-import NavigationGroup from "../ic-navigation-group/readme.md";
+import { Meta, Story, Canvas, ArgsTable, Description } from '@storybook/addon-docs';
```
Comment (GCHQ-Developer-847) [inline packages/web-components/src/components/ic-top-navigation/ic-top-navigation.stories.mdx:1]: Looks like some of the formatting in this file has changed - running 'npm run prettier:fix' should fix it 🙂

**Review by GCHQ-Developer-847 — CHANGES_REQUESTED** [review]
Looks great! Just a couple of things from me

**Review by GCHQ-Developer-847 — APPROVED** [review]

**Review by ad9242 — APPROVED** [review]

**Review by DBD324 — CHANGES_REQUESTED** [review]
When trying different combinations of titles in the new version I seem to still get some strange hyphenation behaviour. On the image below I used a title of some long words and some short ones. Oddly, the first word that wraps onto three lines doesn't use a hyphen between line 1 and 2, but does between line 2 and 3. 

image

**Comment by ad9242** [comment]
> When trying different combinations of titles in the new version I seem to still get some strange hyphenation behaviour. On the image below I used a title of some long words and some short ones. Oddly, the first word that wraps onto three lines doesn't use a hyphen between line 1 and 2, but does between line 2 and 3.
> 
> image

I think it might be a quirk of the browser. I just tried outside of the component with different text and get this behaviour: 
image

**Comment by DBD324** [comment]
Is there a method to truncate instead of wrap? Worried about the effect of long title on small screen devices where the wrapping would push the page content down the page. Truncation and tooltip may be a handy solution here for pointer users.

**Review by DBD324 — APPROVED** [review]
Thanks for exploring the wrapping and hyphenation behavior. Looks good.

**Pull request merged**